spec/latest/core/core.jsonld

```diff
@@ -139,11 +139,11 @@
       "vs:term_status": "testing"
     },
     {
-      "@id": "hydra:statusCodes",
+      "@id": "hydra:possibleStatus",
       "@type": "hydra:Link",
       "label": "status codes",
       "comment": "Additional information about status codes that might be returned by the Web API",

```

Comment:
Done and amended as 0b3e12fe2a7609fb5094888a028a495b637a3e3c.

I did dropped ”description of” though (use / mentions) and made it singular, in accordance with other properties:

> A status that might be returned by the Web API (other statuses should be expected and properly handled as well)